xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/doc/XWikiDocument.java

```diff
@@ -202,6 +202,8 @@
 import com.xpn.xwiki.web.Utils;
 import com.xpn.xwiki.web.XWikiRequest;
 
+import static org.apache.commons.lang3.exception.ExceptionUtils.getRootCauseMessage;
```

Comment:
Do we have a rule about importing static methods @vmassol ? I know we allow this for tests, but I don't remember if we allow this for actual code.